jms/src/main/scala/akka/stream/alpakka/jms/impl/JmsConnector.scala

```diff
@@ -270,16 +271,17 @@ private[jms] trait JmsConnector[S <: JmsSession] extends TimerGraphStageLogic wi
     eventualConnection.map(closeConnection).map(_ => Done)
 
   protected def closeSessions(): Unit = {
-    jmsSessions.foreach(s => closeSession(s))
+    jmsSessions.foreach(closeSession)
     jmsSessions = Seq.empty
   }
 
   protected def closeSessionsAsync(): Future[Unit] = {
     val closing = Future
       .sequence {
-        jmsSessions.map(s => Future(closeSession(s)))
+        jmsSessions.map(s => Future { closeSession(s) })
```

Comment:
I have a strong preference for curly braces around `Future.apply` calls to make them visually stand out.